recorder/src/main/cpp/profiler.cc

```diff
@@ -36,6 +37,7 @@ void Profiler::handle(int signum, siginfo_t *info, void *context) {
         if (thread_info != nullptr) {//TODO: increment a counter here to monitor freq of this, it could be GC thd or compiler-broker etc
             ctx_tracker = &(thread_info->ctx_tracker);
             if (! ctx_tracker->should_record()) {
+                s_c_cpu_samp_drop_norec.inc();
```

Comment:
I was unsure too. Kept it because it can help someone trying to figure out why profiler doesn't show hotspots they expect. But these things have overhead, so if you too think its not needed, I'll just kill it.

Do confirm here.